images/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM registry.access.redhat.com/ubi9/openjdk-11:1.21-1 AS builder
+
+ARG GIT_REVISION=main
```

Comment:
it seems we don't need to manually do git clone. See https://gitlab.cee.redhat.com/project-ncl/konflux-integration/-/blob/main/src/main/image/Dockerfile?ref_type=heads

Reply:
Cool.